npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/version_resolver.rb

```diff
@@ -436,6 +450,25 @@ def run_npm_checker(path:, version:)
           end
         end
 
+        def run_npm7_checker(version:)
+          SharedHelpers.run_shell_command(
+            "npm install #{version_install_arg(version: version)} --package-lock-only --dry-run=true --ignore-scripts"
+          )
+          nil
+        rescue SharedHelpers::HelperSubprocessFailed => e
+          raise if e.message.match?(NPM7_PEER_DEP_ERROR_REGEX)
+        end
+
+        def version_install_arg(version:)
+          source = dependency.requirements.find { |req| req[:source] && req[:source][:type] == "git" }
+
+          if source && source[:type] == "git"
+            "#{dependency.name}@#{source.url}##{version}"
```

Comment:
Looks like you maybe didn't push changes for this? 🤔

Reply:
Wondering if it's worth setting up a test cases for this? Seems like it's not tested atm but this code-path will run for all git dependencies in npm 7 projects

Reply:
🤔 I thought I pulled in your change